test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -648,7 +648,7 @@ private void TestInvalidTargetFramework(string testName, string targetFramework,
 
         [Theory]
         [InlineData("netcoreapp2.2")]
-        [InlineData("netstandard2.1")]
+        [InlineData("netstandard2.3")]
```

Comment:
what was this about?

Reply:
Looks like a bug, there's still no netstandard2.1 TFM. I think it's because the bundled versions aren't right.

Reply:
Filled https://github.com/dotnet/sdk/issues/1632